src/API/WCS/ConnectionService.php

```diff
@@ -12,26 +17,38 @@
  *
  * @package Automattic\WooCommerce\GoogleListingsAndAds\API\WCS
  */
-class ConnectionService implements Service {
+class ConnectionService implements Service, ContainerAwareInterface {
+
+	use ContainerAwareTrait;
 
 	/**
 	 * Return an array of feature flags from the external API.
 	 *
 	 * @return array
+	 *
+	 * @throws Exception When an API error occurs.
 	 */
 	public function get_features(): array {
-		// TODO: Replace hardcoded values with request to external API.
-		$features = [
-			'version'  => gmdate( 'c' ),
-			'features' => [
-				'google_tag_gateway' => [
-					'enabled'    => true,
-					'default'    => true,
-					'percentage' => 100,
-				],
-			],
-		];
-
-		return apply_filters( 'woocommerce_gla_wcs_feature_flags', $features );
+		try {
+			/** @var Client $client */
+			$client   = $this->container->get( Client::class );
+			$result   = $client->get( $this->get_connection_url() );
```

Comment:
Should check `200 === $result->getStatusCode()` before process further. See https://github.com/woocommerce/google-listings-and-ads/blob/develop/src/API/Google/Connection.php#L107-L130

Reply:
This has been updated